src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,5 +1,7 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
```

Comment:
this comment section is a JavaDoc of the `login()` method which should be right above it